cogs/bot.py

```diff
@@ -513,8 +513,8 @@ async def pick(self, ctx, *, name: str):
         )
         await self.bot.redis.hdel("db:member", ctx.author.id)
 
-        await ctx.send(
-            f"Congratulations on entering the world of pokémon! {species} is your first pokémon. Type `{ctx.clean_prefix}info` to view it!"
+        await ctx.reply(
+            f"Congratulations on entering the world of pokémon! {species} is your first pokémon. Type `{ctx.clean_prefix}info` to view it!", mention_author=False
```

Comment:
It's ok to make this ping, since this message will only ever be sent once to someone, having it ping makes sense.